core/src/main/resources/error/error-classes.json

```diff
@@ -687,6 +647,63 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_FORMAT" : {
```

Comment:
Do we want this to be generic or specific to numbers.
I'm specifically wondering about DATE FORMAT, INTERVAL FORMAT, .. What have we/will we do with these?

Reply:
Basically: Should we use INVALID_NUMBER_FORMAT, INVALID_DATETIME_FROMAT, .... or just this and bring them all in?

Reply:
> Do we want this to be generic or specific to numbers.

I want to believe that we can make it as a generic error class. I have already merged `LIKE ...` errors into it.

I think there is the common part where we declare that the format is invalid and show it.